src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,79 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+type Props = {
+  totalItems: number;
+  itemsPerPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  totalItems,
+  itemsPerPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(totalItems / itemsPerPage);
+  const pageNumbers = [];
+
+  for (let i = 1; i <= totalPages; i += 1) {
+    pageNumbers.push(i);
+  }
+
+  return (
+    <>
+      <ul className="pagination">
+        <li className={cn('page-item', { disabled: currentPage === 1 })}>
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            aria-disabled={currentPage === 1}
+            onClick={() => {
+              if (currentPage > 1) {
+                onPageChange(currentPage - 1);
+              }
+            }}
```

Comment:
Consider using a separate handler function for the `onClick` event. Inline functions in JSX can lead to performance issues because they are recreated on every render.